modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -377,3 +388,117 @@ Scanner output rules are used in daemon/socket mode.
 . A file size limit applies both to background jobs and to file uploads.
 . After installing ClamAV and the related tools, you will have two configuration files: `/etc/freshclam.conf` and `/etc/clamd.d/scan.conf`. 
 . We recommend that you enable verbose logging in both `clamd.conf` and `freshclam.conf` until you get any kinks with your ClamAV installation worked out
+
+
+== ICAP
+
+{icap-url}[ICAP] is an open standard supported by many antivirus products. With the release of the _Anti-Virus_ app 1.0.0, other virus scanners beside ClamAV can be used via ICAP if you are running it on an ownCloud Enterprise Edition. Currently the only supported virus scanners are _Kaspersky ScanEngine_ and _McAfee Antivirus_ although far more products might simply work.
+
+=== Installation
+
+- Install the Anti-Virus app 1.0.0 from the {anti-virus-app-url}[ownCloud marketplace]. Alternatively, use this occ command:
```

Comment:
remove the release info
also use {oc-marketplace-url}/apps/files_antivirus[Anti-Virus app]

Reply:
Done.